tools/sync_test_server/integration-test-command-server.js

```diff
@@ -119,10 +95,10 @@ function waitForPortToBeReady(attempts, onSuccess, onError) {
     });
 }
 
-function doStartRealmObjectServer(onSuccess, onError) {
-    temp.mkdir('ros', function(err, path) {
-        if (!err) {
-            winston.info("command-server: Starting sync server in ", path);
+function startRealmObjectServer(onSuccess, onError) {
+    stopRealmObjectServer(() => {
```

Comment:
You are on your way to become a Javascript backend developer 😄

Reply:
🙈

Reply:
👀